src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/ReadPosRankSumTestUnitTest.java

```diff
@@ -158,4 +148,28 @@ public void testReadPos_Raw(){
         Assert.assertTrue(annotatePastEndRaw.isEmpty());
         Assert.assertTrue(annotatePastEndNonRaw.isEmpty());
     }
+
+    @DataProvider(name = "dataIsUsableRead")
+    private Object[][] dataIsUsableRead(){
+        return new Object[][]{
+                {"20M6D2M", 10, 1, 0, true},
+                {"20M6D2M", 10, 1, 29, false},
```

Comment:
Can you add one more test case where the read is usable to cover a position after the deletion?  I think  {"20M6D2M", 10, 1, 27, true} would do it, right?  That should look like our test case from GATK3:
![image](https://cloud.githubusercontent.com/assets/6578548/21232780/dd40a608-c2ba-11e6-97e4-5d06042422cd.png)